src/main/java/uk/gov/hmcts/probate/model/ccd/caveat/request/CaveatData.java

```diff
@@ -2,18 +2,24 @@
 
 import lombok.Builder;
 import lombok.Data;
+import uk.gov.hmcts.probate.model.ApplicationType;
 import uk.gov.hmcts.probate.model.ccd.caveat.CavAddress;
 import uk.gov.hmcts.probate.model.ccd.caveat.CavFullAliasName;
 import uk.gov.hmcts.probate.model.ccd.raw.CollectionMember;
+import uk.gov.hmcts.probate.model.ccd.raw.Document;
 import uk.gov.hmcts.probate.model.ccd.raw.UploadDocument;
 
 import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
 
 @Builder
 @Data
 public class CaveatData {
 
+    private final ApplicationType cavApplicationType;
```

Comment:
Since we are in Caveat specific class already, can we get rid of `cav` prefixes?